extensions/wikia/PortableInfobox/templates/PortableInfoboxWrapper.mustache

```diff
@@ -1 +1 @@
-<aside class="portable-infobox">{{{content}}}</aside>
\ No newline at end of file
+<aside class="portable-infobox {{{theme}}}">{{{content}}}</aside>
```

Comment:
I wonder if passing {{{classes}}} would be clearer and help avoid the whitespace in markup below